tests_integration/__tests__/__snapshots__/loglevel.js.snap

```diff
@@ -0,0 +1,37 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`do not show logs with --loglevel silent (stderr) 1`] = `""`;
+
+exports[`do not show logs with --loglevel silent (stdout) 1`] = `""`;
+
+exports[`do not show logs with --loglevel silent (write) 1`] = `Array []`;
+
+exports[`do not show warnings with --loglevel error (stderr) 1`] = `
+"[error] No matching files. Patterns tried: not-found.js !**/node_modules/** !./node_modules/**
+"
+`;
+
+exports[`do not show warnings with --loglevel error (stdout) 1`] = `""`;
+
+exports[`do not show warnings with --loglevel error (write) 1`] = `Array []`;
+
+exports[`show all logs with --loglevel debug (stderr) 1`] = `
+"[warn] Ignored unknown option: --unknown-option
+[debug] normalized argv: {\\"_\\":[\\"not-found.js\\"],\\"bracket-spacing\\":false,\\"color\\":true,\\"debug-check\\":false,\\"debug-print-doc\\":false,\\"flow-parser\\":false,\\"insert-pragma\\":false,\\"jsx-bracket-same-line\\":false,\\"list-different\\":false,\\"require-pragma\\":false,\\"semi\\":false,\\"single-quote\\":false,\\"stdin\\":false,\\"use-tabs\\":false,\\"version\\":false,\\"with-node-modules\\":false,\\"write\\":false,\\"loglevel\\":\\"debug\\",\\"parser\\":\\"unknown-parser\\",\\"ignore-path\\":\\".prettierignore\\",\\"config-precedence\\":\\"cli-override\\"}
```

Comment:
Is this test prone to breaking?

Reply:
It seems it'd be better to use `expect(...).toMatch(/[loglevel]/)` in this case.